client/sites/settings/index.tsx

```diff
@@ -140,8 +140,7 @@ export default function () {
 		'/sites/settings/v2/:site/:feature?',
 		siteSelection,
 		navigation,
-		dashboardBackportSiteSettings,
-		siteDashboard( SETTINGS_SITE ),
+		redirectToSiteSettingsNewUrl,
 		makeLayout,
 		clientRender
```

Comment:
I'm surprised to see `makeLayout` and `clientRender` appear after `redirectToSiteSettingsNewUrl` in the controller list.

`redirectToSiteSettingsNewUrl` unconditionally redirects, so it doesn't seem like the last two controllers are used ever? But maybe it is need to prevent some glitchy rendering while the redirect is happening? 🤷

Reply:
Thanks, I just forgot to remove them 🤦 Removed!